### ISSUE TITLE
Clarify statement in progressbar role definition

### ISSUE THREAD

**Issue by dev-nicolaos** [w3c/aria#1654]
In _[5.4 > progressbar](https://www.w3.org/TR/wai-aria-1.2/#progressbar)_

> If the progressbar is describing the loading progress of a particular region of a page, the author SHOULD use aria-describedby to point to the status, and set the aria-busy attribute to true on the region until it is finished loading.

This statement leaves it somewhat unclear whether the author should put the `aria-describedby` attribute on the region of the page that is loading and point it to the progressbar or vice-versa. Contrast this with the instructions for `aria-busy`, where it is made clear that attribute should be placed on the region and guidance is given on what to do regarding the attribute when the loading has finished.

TLDR: clarify which element `aria-describedby` should go on and which element it should point to in this case, and what should be done when the loading finishes.

Related: https://github.com/mdn/content/pull/10707

**Comment by scottaohara** [comment]
this is meant to indicate that `aria-describedby` is used on the section of content that is in the loading state, as the progress bar is conveying a "status".  

Seems like this could be an easy editorial update for someone to take on?
